interested-parties.md

```diff
@@ -15,6 +15,7 @@ Cerence | Hatice Ozguven
 Cloud Native Architects & Qwinix | Leonardo Murillo
 CNCF | Katie Gamanji
 Codefresh | Dan Garfield
+comforte AG | Piotr Zaniewski (Piotr1215)
```

Comment:
@Piotr1215 Generally re: DCO, I found the the following useful.  Thank you!

@scottrigby wrote in https://github.com/gitops-working-group/gitops-working-group/pull/117
>  just need to amend your commit to add your DCO signoff
>
> PS, you may be interested in this fun plugin I made to help with this when using the GitHub UI 😸 https://github.com/scottrigby/dco-gh-ui/
>
> I made this because this issue eventually happens to almost everyone contributing to CNCF/LF projects that require DCO.

Reply:
@Piotr1215 To facilitate future merging, would you kindly edit by using the syntax based on examples at
• https://github.com/gitops-working-group/gitops-working-group/pull/170/files
and
• https://github.com/gitops-working-group/gitops-working-group/pull/171/files
?

For example, you would add:
1. title row and 2. horizontal separators near the top as:
```
Organization | Person | GitHub Username
-- | -- | --
```
3. vertical separators as:
```suggestion
comforte AG | Piotr Zaniewski | Piotr1215
```
Thank you!